### ISSUE TITLE
Upload the LSUN dataset link.

### ISSUE THREAD

**Issue by faraz6313** [GitBoSun/roomnet#4]
Sir, Can you please upload the link of  LSUN dataset which you use in your code.

**Comment by Gabri3l** [comment]
The general reference for these resources is this one: [http://lsun.cs.princeton.edu/challenge/2015/roomlayout/data/](http://lsun.cs.princeton.edu/challenge/2015/roomlayout/data/). Here you can find the dataset, ground truths, and metadata.

**Comment by Wap8CN** [comment]
@DRAhmadFaraz @Gabri3l @GitBoSun I can't open this link. It returns 404 Not Found. Would you kindly share the image.zip file(2G) in Google Drive？

**Comment by Gabri3l** [comment]
For future reference please create a copy of this dataset for yourself, it is available here for the time being but I'm not sure how long for.

[LSUN](https://console.cloud.google.com/storage/browser/roomnet_datasets)

**Comment by lebeli** [comment]
Thank you for the link. Could you provide access to the ground truth data as well? It is sadly not available either.

EDIT: It would be very intersting to get the original full resolution images, could I ask for them as well?

**Comment by Gabri3l** [comment]
[LSUN + all .mat files](https://drive.google.com/drive/folders/1bEmt8LNfepTx8ekwUb5vBBrsKnZA6kM9?usp=sharing)

There you go everyone, I was able to go back and find everything...enjoy 🎉

**Comment by linzhi-li** [comment]
> [LSUN + all .mat files](https://drive.google.com/drive/folders/1bEmt8LNfepTx8ekwUb5vBBrsKnZA6kM9?usp=sharing)
> 
> There you go everyone, I was able to go back and find everything...enjoy 🎉

where did you get the access to the .mat files? Did the challenge data provider offer the whole .mat files or you label the data yourselves? Thanks~

**Comment by Gabri3l** [comment]
The challenge provided the `.mat` files

**Comment by linzhi-li** [comment]
> The challenge provided the `.mat` files

Are the files you offered the whole data and files from the challenge? I will rely on the files you shared to reproduce some experiments, thus I am trying collecting the whole training data as possible. Thanks~

**Comment by Gabri3l** [comment]
Yeah that's what was available on the challenge page when it was still up. Also beware this repo has quite a few bugs especially in data prepping. So keep an eye out.

**Comment by linzhi-li** [comment]
> Yeah that's what was available on the challenge page when it was still up. Also beware this repo has quite a few bugs especially in data prepping. So keep an eye out.

Thank you again~

**Comment by zhangjingxian1998** [comment]
> Yeah that's what was available on the challenge page when it was still up. Also beware this repo has quite a few bugs especially in data prepping. So keep an eye out.

Thank you very much, sir!
